coordinator/src/test/scala/filodb.coordinator/queryplanner/HighAvailabilityPlannerSpec.scala

```diff
@@ -31,8 +31,8 @@ class HighAvailabilityPlannerSpec extends AnyFunSpec with Matchers {
   private val dsRef = dataset.ref
   private val schemas = Schemas(dataset.schema)
 
-  private val routingConfigString = "routing {\n  remote {\n   " +
-    " http {\n      endpoint = localhost\n timeout = 10000\n    }\n  }\n}"
+  private val routingConfigString = "routing {\n  remote {\n    http {\n      endpoint = localhost\n      timeout = 10000\n" +
+    "      client.sttp-backend.factory = filodb.query.exec.AsyncHttpClientBackendFactory\n    }\n  }\n}"
```

Comment:
Use `"""` format for multi line strings

Reply:
This change is no more required. Reverted